build.gradle

```diff
@@ -176,6 +176,9 @@ dependencies {
     compile group: 'com.sun.mail', name: 'mailapi', version: '1.6.0'
     compile group: 'commons-lang', name: 'commons-lang', version: '2.6'
 
+    //simplifies handling of exceptions with Java CompletableFuture
+    compile group: 'org.zalando', name: 'faux-pas', version: '0.6.0'
```

Comment:
handling of Exceptions in CompletableFuture involves lots of boilerplate. This library can help